akka-docs/src/test/java/jdocs/stream/operators/converters/ToFromJavaIOStreams.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package jdocs.stream.operators.converters;
+
+import akka.NotUsed;
+import akka.actor.ActorSystem;
+import akka.stream.IOResult;
+import akka.stream.Materializer;
+import akka.stream.javadsl.Flow;
+import akka.stream.javadsl.Sink;
+import akka.stream.javadsl.Source;
+import akka.stream.javadsl.StreamConverters;
+import akka.testkit.javadsl.TestKit;
+import akka.util.ByteString;
+import jdocs.AbstractJavaTest;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.nio.charset.Charset;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import static org.junit.Assert.assertEquals;
+
+public class ToFromJavaIOStreams extends AbstractJavaTest {
+
+  static ActorSystem system;
+
+  @BeforeClass
+  public static void setup() {
+    system = ActorSystem.create("ToFromJavaIOStreams");
+  }
+
+  @AfterClass
+  public static void tearDown() {
+    TestKit.shutdownActorSystem(system);
+    system = null;
+  }
+
+  @Test
+  public void demonstrateFromJavaIOStreams()
+      throws InterruptedException, ExecutionException, TimeoutException {
+    // #tofromJavaIOStream
+    Charset charset = Charset.defaultCharset();
+    byte[] bytes = "Some random input".getBytes(charset);
+
+    ByteArrayInputStream inputStream = new ByteArrayInputStream(bytes);
```

Comment:
I think it might be more concise if you hide the setup from the shown code. Just showing some 
```java
java.util.InputStream inputStream = // ???
java.util.OutputStream outStream = // ???
```

Reply:
Hmmm, let me see how the page looks like with the suggested change...